book/docs/1_introduction_to_data_engineering/1.4_further_more.md

```diff
@@ -36,5 +36,3 @@
     - 분석 프로세스 최적화
     - 대시보드 개발 및 운영
     - 데이터 파이프라인 개발 및 운영
-
-
```

Comment:
제 환경에서 빌드 오류나고 있던 원인이 이 부분이었는데요.. (해당 빈 line을 제거하니까 빌드 오류가 해결됐습니다!)
왜 이 부분 때문에 오류가 난건지는 파악을 못했습니다🥲

<details><summary>참고 - 발생했던 오류 로그</summary>

```
Running Jupyter-Book v0.15.1                                                                               
Source Folder: /Users/user/Documents/data-engineering-for-everybody/book                                   
Config Path: /Users/user/Documents/data-engineering-for-everybody/book/_config.yml                         
Output Path: /Users/user/Documents/data-engineering-for-everybody/book/_build/html                         
Running Sphinx v4.5.0                                                                                      
[etoc] Changing master_doc to 'intro'                                                                      
myst v0.18.1: MdParserConfig(commonmark_only=False, gfm_only=False, enable_extensions=['html_image'], disab
le_syntax=[], all_links_external=False, url_schemes=['mailto', 'http', 'https'], ref_domains=None, highligh
t_code_blocks=True, number_code_blocks=[], title_to_header=False, heading_anchors=None, heading_slug_func=N
one, footnote_transition=True, words_per_minute=200, sub_delimiters=('{', '}'), linkify_fuzzy_links=True, d
math_allow_labels=True, dmath_allow_space=True, dmath_allow_digits=True, dmath_double_inline=False, update_
mathjax=True, mathjax_classes='tex2jax_process|mathjax_process|math|output_area')                          
myst-nb v0.17.2: NbParserConfig(custom_formats={}, metadata_key='mystnb', cell_metadata_key='mystnb', kerne
l_rgx_aliases={}, execution_mode='off', execution_cache_path='', execution_excludepatterns=[], execution_ti
meout=30, execution_in_temp=False, execution_allow_errors=False, execution_raise_on_error=False, execution_
show_tb=False, merge_streams=False, render_plugin='default', remove_code_source=False, remove_code_outputs=
False, code_prompt_show='Show code cell {type}', code_prompt_hide='Hide code cell {type}', number_source_li
nes=False, output_stderr='show', render_text_lexer='myst-ansi', render_error_lexer='ipythontb', render_imag
e_options={}, render_figure_options={}, render_markdown_format='commonmark', output_folder='build', append_
css=True, metadata_to_fm=False)                                                                            
Using jupyter-cache at: /Users/user/Documents/data-engineering-for-everybody/book/_build/.jupyter_cache    
building [mo]: targets for 0 po files that are out of date                                                 
building [html]: targets for 63 source files that are out of date                                          
updating environment: [new config] 63 added, 0 changed, 0 removed                                          
reading sources... [  9%] docs/1_introduction_to_data_engineering/1.4_further_more                         
Exception occurred:                                                                                        
  File "/Users/user/opt/anaconda3/lib/python3.9/site-packages/markdown_it/rules_block/state_block.py", line
 137, in isEmpty                                                                                           
    return (self.bMarks[line] + self.tShift[line]) >= self.eMarks[line]                                    
IndexError: list index out of range                                                                        
The full traceback has been saved in /var/folders/fb/xtftd5y96nl3xc10kvb6cd4w0000gn/T/sphinx-err-rfafr17r.l
og, if you want to report the issue to the developers.                                                     
Please also report this if it was a user error, so that a better error message can be provided next time.  
A bug report can be filed in the tracker at <https://github.com/sphinx-doc/sphinx/issues>. Thanks!         
Traceback (most recent call last):                                                                         
  File "/Users/user/opt/anaconda3/lib/python3.9/site-packages/jupyter_book/sphinx.py", line 171, in build_s
phinx
    app.build(force_all, filenames)
  File "/Users/user/opt/anaconda3/lib/python3.9/site-packages/sphinx/application.py", line 330, in build
    self.builder.build_update()
  File "/Users/user/opt/anaconda3/lib/python3.9/site-packages/sphinx/builders/__init__.py", line 286, in bu
ild_update
    self.build(to_build,
  File "/Users/user/opt/anaconda3/lib/python3.9/site-packages/sphinx/builders/__init__.py", line 300, in bu
ild
    updated_docnames = set(self.read())
  File "/Users/user/opt/anaconda3/lib/python3.9/site-packages/sphinx/builders/__init__.py", line 407, in re
ad
    self._read_serial(docnames)
  File "/Users/user/opt/anaconda3/lib/python3.9/site-packages/sphinx/builders/__init__.py", line 428, in _r
ead_serial
    self.read_doc(docname) 
  File "/Users/user/opt/anaconda3/lib/python3.9/site-packages/sphinx/builders/__init__.py", line 468, in re
ad_doc
    doctree = read_doc(self.app, self.env, self.env.doc2path(docname))
  File "/Users/user/opt/anaconda3/lib/python3.9/site-packages/sphinx/io.py", line 181, in read_doc
    pub.publish()
  File "/Users/user/opt/anaconda3/lib/python3.9/site-packages/docutils/core.py", line 217, in publish
    self.document = self.reader.read(self.source, self.parser,
  File "/Users/user/opt/anaconda3/lib/python3.9/site-packages/sphinx/io.py", line 101, in read
    self.parse()
  File "/Users/user/opt/anaconda3/lib/python3.9/site-packages/docutils/readers/__init__.py", line 78, in pa
rse
    self.parser.parse(self.input, document)
  File "/Users/user/opt/anaconda3/lib/python3.9/site-packages/myst_nb/sphinx_.py", line 86, in parse
    return super().parse(inputstring, document)
  File "/Users/user/opt/anaconda3/lib/python3.9/site-packages/myst_parser/parsers/sphinx_.py", line 69, in 
parse
    parser.render(inputstring)
  File "/Users/user/opt/anaconda3/lib/python3.9/site-packages/markdown_it/main.py", line 275, in render
    return self.renderer.render(self.parse(src, env), self.options, env)
  File "/Users/user/opt/anaconda3/lib/python3.9/site-packages/markdown_it/main.py", line 260, in parse
    self.core.process(state)
  File "/Users/user/opt/anaconda3/lib/python3.9/site-packages/markdown_it/parser_core.py", line 33, in proc
ess
    rule(state)
  File "/Users/user/opt/anaconda3/lib/python3.9/site-packages/markdown_it/rules_core/block.py", line 14, in
 block
    state.md.block.parse(
  File "/Users/user/opt/anaconda3/lib/python3.9/site-packages/markdown_it/parser_block.py", line 107, in pa
rse
    self.tokenize(state, state.line, state.lineMax)
  File "/Users/user/opt/anaconda3/lib/python3.9/site-packages/markdown_it/parser_block.py", line 77, in tok
enize
    if rule(state, line, endLine, False):
  File "/Users/user/opt/anaconda3/lib/python3.9/site-packages/markdown_it/rules_block/list.py", line 264, i
n list_block
    state.md.block.tokenize(state, startLine, endLine)
  File "/Users/user/opt/anaconda3/lib/python3.9/site-packages/markdown_it/parser_block.py", line 77, in tok
enize
    if rule(state, line, endLine, False):
  File "/Users/user/opt/anaconda3/lib/python3.9/site-packages/markdown_it/rules_block/list.py", line 272, i
n list_block
    prevEmptyEnd = (state.line - startLine) > 1 and state.isEmpty(state.line - 1)
  File "/Users/user/opt/anaconda3/lib/python3.9/site-packages/markdown_it/rules_block/state_block.py", line
 137, in isEmpty
    return (self.bMarks[line] + self.tShift[line]) >= self.eMarks[line]
IndexError: list index out of range

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Users/user/opt/anaconda3/bin/jupyter-book", line 8, in <module>
    sys.exit(main())
  File "/Users/user/opt/anaconda3/lib/python3.9/site-packages/click/core.py", line 1128, in __call__
    return self.main(*args, **kwargs)
  File "/Users/user/opt/anaconda3/lib/python3.9/site-packages/click/core.py", line 1053, in main
    rv = self.invoke(ctx)
  File "/Users/user/opt/anaconda3/lib/python3.9/site-packages/click/core.py", line 1659, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/Users/user/opt/anaconda3/lib/python3.9/site-packages/click/core.py", line 1395, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/Users/user/opt/anaconda3/lib/python3.9/site-packages/click/core.py", line 754, in invoke
    return __callback(*args, **kwargs)
  File "/Users/user/opt/anaconda3/lib/python3.9/site-packages/jupyter_book/cli/main.py", line 317, in build
    builder_specific_actions(
  File "/Users/user/opt/anaconda3/lib/python3.9/site-packages/jupyter_book/cli/main.py", line 525, in build
er_specific_actions
    raise RuntimeError(_message_box(msg, color="red", doprint=False)) from result
RuntimeError: 
===============================================================================

There was an error in building your book. Look above for the cause.

===============================================================================
```

</details>